src/components/Divider/native/Divider.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Divider = styled.View`
+  borderTopWidth: ${(p) => p.vertical ? 0 : p.thickness};
```

Comment:
turn camelCase css properties to kebab-case

Reply:
done